src/strategies/dkuma-breeder-staked/index.ts

```diff
@@ -0,0 +1,44 @@
+import { BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller } from '../../utils';
+// import { multicall } from '../../utils';
+import { getAddress } from '@ethersproject/address';
+
+export const author = 'theKdev9';
+export const version = '0.1.0';
+
+interface walletInfoInt {
+  amount: BigNumberish;
+  enteredAt: BigNumberish;
+  rewardTaken: BigNumberish;
+  rewardTakenActual: BigNumberish;
+  bag: BigNumberish;
+}
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+): Promise<Record<string, number>> {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+
+  const multi = new Multicaller(network, provider, [options.dbreeder.abi], {
+    blockTag
+  });
+  addresses.forEach((address) =>
+    multi.call(getAddress(address), options.dbreeder.address, 'stakeInfo', [
+      address
+    ])
+  );
+  const result: Record<string, walletInfoInt> = await multi.execute();
+
+  return Object.fromEntries(
+    Object.entries(result).map(([address, walletInfo]) => [
+      address,
+      parseFloat(formatUnits(walletInfo.amount, options.decimals))
+    ])
```

Comment:
Hey @KumaInuCommunity You can use https://snapshot.org/#/strategy/contract-call strategy instead. did you try it?

Reply:
That's right, hadn't seen it. Thank you.
Now we are using 3 different strategies, all of them being default strategies of yours. Is it fine if we create a single strategy to call all of them so the input options can be simplified?

Reply:
@KumaInuCommunity We expect strategies to be unique :) If it is possible to achieve something with 3 strategies, Better to keep them separate so our cache systems work perfectly, it can resolve much faster and we can reduce code duplication.

Reply:
Makes sense, let´s keep it as it is now. Thank you.

Reply:
Makes sense, let's keep it as it is now. Thank you.